.github/workflows/python-builder.yml

```
@@ -0,0 +1,220 @@
+name: Build python package
+
+on:
+  workflow_dispatch:
+    inputs:
+      VERSION:
+        description: 'Python version to build and upload'
+        default: '3.9.6'
```

time to update the default? 😉
```suggestion
        default: '3.9.9'
```